server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/newyorkcoin/NewyorkcoinExtension.java

```diff
@@ -94,32 +94,7 @@ public IRateSource createRateSource(String sourceLogin) {
             StringTokenizer st = new StringTokenizer(sourceLogin,":");
             String rsType = st.nextToken();
 
-            if ("coingecko".equalsIgnoreCase(rsType)) {
-                String preferredFiatCurrency = st.hasMoreTokens() ? st.nextToken().toUpperCase() : FiatCurrency.USD.getCode();
-                return new CoinGeckoRateSource(preferredFiatCurrency);
-            } else if ("coinmarketcap".equalsIgnoreCase(rsType)) {
-                String preferredFiatCurrency = FiatCurrency.USD.getCode();
-                String apiKey = null;
-                if (st.hasMoreTokens()) {
-                    preferredFiatCurrency = st.nextToken().toUpperCase();
-                }
-                if (st.hasMoreTokens()) {
-                    apiKey = st.nextToken();
-                }
-                return new CoinmarketcapRateSource(apiKey, preferredFiatCurrency);
-            } else if ("nycfix".equalsIgnoreCase(exchangeType)) {
-                BigDecimal rate = BigDecimal.ZERO;
-                if (st.hasMoreTokens()) {
-                    try {
-                        rate = new BigDecimal(st.nextToken());
-                    } catch (Throwable e) {
-                    }
-                }
-                String preferedFiatCurrency = FiatCurrency.USD.getCode();
-                if (st.hasMoreTokens()) {
-                    preferedFiatCurrency = st.nextToken().toUpperCase();
-                }
-                return new FixPriceRateSource(rate,preferedFiatCurrency);
```

Comment:
you need to keep "nycfix" that you are adding in this pull request, just remove "coingecko" and "coinmarketcap" that are already present in other extensions

Reply:
ahh my mistake and I can't locate template.png in the code

Reply:
appologies, please use `template.svg` and export the PNG so it has the same style and dimensions like other logos, thanks

Reply:
Ahh ok, I did that, I presumed that is what you meant when I couldn't locate the png template